docs/deployment/README.md

```diff
@@ -346,12 +360,15 @@ If you've lost track of the needed redirect URIs, they can be located on the [AP
 
 ### Install dashboard
 
-To install the dashboard clone the Cumulus-dashboard repository into the root deploy directory and install dependencies with `npm install`:
+To install the dashboard clone the Cumulus-dashboard repository into the root deploy directory and install dependencies with `yarn install`:
 
     $ git clone https://github.com/nasa/cumulus-dashboard
     $ cd cumulus-dashboard
     $ nvm use
-    $ npm install
+    $ yarn install
+
+If you do not have the correct version of node installed, replace `nvm use` with `nvm install $(cat .nvmrc)` in the above example.
```

Comment:
I think you would want to do the `nvm install` *and* then the `nvm use`?

Reply:
The nvm install seems to  also activate the environment.